src/System.Security.Cryptography.ProtectedData/src/System.Security.Cryptography.ProtectedData.builds

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.Security.Cryptography.ProtectedData.csproj" />
```

Comment:
It looks like this project only runs on Windows, correct?If so, please make this line:

```
<Project Include="<csproj>">
   <OSGroup>Windows_NT</OSGroup>
</Project>
```